contracts/wrapper/NameWrapper.sol

```diff
@@ -302,7 +308,8 @@ contract NameWrapper is
             revert RenewalTooShort(bytes32(tokenId));
         }
         if (isWrapped(node)) {
-            (address owner, uint32 oldFuses, ) = getData(uint256(node));
```

Comment:
I think it's better to duplicate the fuse-erasure logic here than to get the data twice. But do we want to consider keeping the fuses if a name is renewed during the grace period, instead?

Reply:
Actually I think the fuses will be kept anyway, because when it is renewed, the name is unexpired and the fuses aren't changed yet, so they are kept. So I can just keep it as getData() since the effect is the same